packages/core/index.test-d.ts

```diff
@@ -268,24 +261,24 @@ const mutableContextMiddleware = {
 };
 
 customCtxHandler = customCtxHandler.use(mutableContextMiddleware);
-expectType<MutableContextHandler>(customCtxHandler);
+expect(customCtxHandler).type.toBe<MutableContextHandler>();
 
 const typeErrorMiddleware = {
 	before: (request: MutableContextRequest) => {
-		// @ts-expect-error
+		// @ts-expect-error!
```

Comment:
TSTyche is able to check messages of error suppressed by `// @ts-expect-error`. This is useful because hides any kind of `// @ts-expect-error`. Since type tests are not executed, there is no way to find the problems.

A message after `// @ts-expect-error` is required. The `!` in this lines tells to ignore that. But maybe I should include the message? Hm..